vm/systemSmartContracts/esdt.go

```diff
@@ -106,16 +105,16 @@ func NewESDTSmartContract(args ArgsNewESDTSmartContract) (*esdt, error) {
 	if len(args.EndOfEpochSCAddress) == 0 {
 		return nil, vm.ErrNilEndOfEpochSmartContractAddress
 	}
+	if check.IfNil(args.StatusHandler) {
+		return nil, vm.ErrNilStatusHandler
+	}
 
 	baseIssuingCost, okConvert := big.NewInt(0).SetString(args.ESDTSCConfig.BaseIssuingCost, conversionBase)
 	if !okConvert || baseIssuingCost.Cmp(big.NewInt(0)) < 0 {
 		return nil, vm.ErrInvalidBaseIssuingCost
 	}
 
-	appStatusHandler := args.StatusHandler
-	if appStatusHandler == nil {
-		appStatusHandler = statusHandler.NewNilStatusHandler()
-	}
+	args.StatusHandler.SetStringValue(common.MetricESDTIssuanceCost, baseIssuingCost.String())
```

Comment:
this will get the initial, old value from the config.
Please call this: `e.getESDTConfig()` on L137+